packages/ngtools/webpack/src/angular_compiler_plugin.ts

```diff
@@ -86,6 +90,11 @@ export interface AngularCompilerPluginOptions {
   // added to the list of lazy routes
   additionalLazyModules?: { [module: string]: string };
 
+  // The ContextElementDependency of correct Webpack compilation.
+  // This is needed when there are multiple Webpack installs.
+  // tslint:disable-next-line:no-any
```

Comment:
can remove the tslint disable here (and the one below on line 140)

Reply:
done